packages/backend-core/src/integrations/youtube/task.ts

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { logger } from '../../logger';
+import { authorize } from './authenticate';
+import { sync_page_of_channels } from './channels';
+
+// Tick is every minute.  Channel sync once per day.
```

Comment:
That seems very slow... is this temporary until we can sync every few minutes or something?